test/types.ts

```diff
@@ -175,14 +173,22 @@ multiset = MultiSet.from({'one': 1});
 /**
  * MultiMap.
  */
-let multimap: MultiMap<number, string> = new MultiMap(Set);
+let multimap: MultiMap<number, string, Set<string>> = new MultiMap(Set);
 multimap.set(45, 'test');
+multimap.get(45).has('test');
 let stringMultimap: MultiMap<string, number> = MultiMap.from({one: 1});
+stringMultimap.get('one').indexOf(1);
+for (const _ of multimap) { }
+for (const _ of multimap.keys()) { }
+for (const _ of multimap.values()) { }
+for (const _ of multimap.entries()) { }
+for (const _ of multimap.containers()) { }
+for (const _ of multimap.associations()) { }
 
 /**
  * PassjoinIndex.
  */
-const passjoinIndex: PassjoinIndex<string> = new PassjoinIndex(leven, 1);
```

Comment:
Why this modification? What's more, `() => 0` should fail the types, no?

Reply:
The primary reason why I removed this is that it seemed unnecessary to have the `leven` dependency only to perform type checking. The same goal can be accomplished by just providing a function with a suitable signature directly.

Since the `PassJoinIndex` constructor takes a function of type `(a: T, b: T) => number`, then it's always legal to pass a function that takes has zero parameters. You have a point though, the test itself should use a function with two parameters as that's the actual use case that we need to verify.